cluster/charts/rook-ceph/values.yaml

```diff
@@ -72,6 +72,8 @@ csi:
   #cephfsGrpcMetricsPort: 9091
   #cephfsLivenessMetricsPort: 9081
   #rbdGrpcMetricsPort: 9090
+  # Enable Ceph Kernel clients on kernel < 4.17 which support quotas for Cephfs
+  #forceCephFSKernelClient: true
```

Comment:
Should we enable this by default? As implemented it appears the default is false. This will hopefully help remediate the problem of using FUSE, which breaks clients during upgrade. That's a serious problem for upgrades currently. Or can you remind me what the solution is for that?

Reply:
@travisn this should not be enabled by default, This is not supported for all the kernel less than 4.17 only a few are supported.

Reply:
Ok, i'll open a separate issue to make sure we get this upgrade issue documented until it's fixed.

Reply:
@Madhu-1 Can you clarify this more? What do you mean "only a few are supported" if kernel is less than 4.17? Does that mean some features aren't supported in cephfs? If so, I want to ask my question again... It seems better to force enable the driver by default and run with some features disabled than to be broken during upgrades.

Reply:
Not all cephfs kernel versions support quota enforcement. If we enable this by default, it will get us over the upgrade issue for sure, but PV space used may violate the requested size. Hence enforcing this by default is not desirable.

Reply:
IMO breaking applications during upgrade is higher severity than quotas not being enforced. They are both serious issues, no doubt, but the default behavior should be the lesser of the two evils. 

Assuming quotas are less severe than being broken during upgrade, we should force the kernel driver by default. Then we can document that if you want quotas enforced, you have two options:
1. Use a kernel >= 4.17
1. Enable the cephfs-fuse option, which means you change `CSI_FORCE_CEPHFS_KERNEL_CLIENT` from the default of `true` back to `false`.

If the user finds quotas are not being enforced, he will consult the documentation and see that he can enable this later.

Reply:
I've pushed a commit to this PR that changes it to be enabled by default. We can discuss more, but this is my assumption that also goes along with docs in the upgrade guide for #4496.

Reply:
we had a lot of discussion about enabling/disabling it by default in ceph-csi. 
more info:
https://github.com/ceph/ceph-csi/pull/623
https://github.com/ceph/ceph-csi/pull/664

cc @humblec